index.html

```diff
@@ -44,7 +44,149 @@ <h5> I can help you build a product ,
       </div>
       <img src="images/Group.png" class="img-fluid ${3|rounded-top,rounded-right,rounded-bottom,rounded-left,rounded-circle,|}" alt="">
   </section> 
-   
+  <section class="works">
+    <h1 class="title container">My Recent Works</h1>
+    <hr>
+     <div class="image container"></div>
+     <h2 class="corner-title container">Multi-Post Stories</h2>
+     <p class="content container">A daily selection of privately personalized
+        reads; no accounts or sign-ups required.
+       has been the industry's standard dummy
+       text ever since the 1500s, when an 
+       unknown printer took a standard dummy text.</p>
+     <div class="lessons container"> 
+     <span class="html">html</span>
+     <span class="css">css</span>
+     <span class="bootstrap boot1">bootstrap</span>
+     <span class="ruby ">ruby</span>
+     </div> 
+         <button type="button" class="see"> See Project</button>
+         <div class="profesional p-1">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-2">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-3">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
```

Comment:
- [ ] The button (e.g., "See project") should follow the button styling: Active, Hover, and Pressed (see design below), and please implement it
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/193013682-4b947d62-c543-4a72-bcba-1bd52f9f5e5b.png)